frontend/src/app/components/patient-info-form/patient-info-form.component.html

```diff
@@ -4,22 +4,36 @@
     <form [formGroup]="form">
       <div class="row">
         <app-form-field label="Jméno *" [invalid]="isControlInvalid('firstName')" [errors]="getControlErrors('firstName')">
-          <input formControlName="firstName" name="firstName" placeholder="Vaše jméno" required maxlength="64">
+          <input formControlName="firstName" name="firstName" placeholder="Vaše jméno" required maxlength="256">
         </app-form-field>
 
         <app-form-field label="Příjmení *" [invalid]="isControlInvalid('lastName')" [errors]="getControlErrors('lastName')">
-          <input formControlName="lastName" name="lastName" placeholder="Vaše příjmení" required maxlength="64">
+          <input formControlName="lastName" name="lastName" placeholder="Vaše příjmení" required maxlength="256">
         </app-form-field>
       </div>
 
       <div class="row">
-        <app-form-field label="Rodné číslo *" note="Rodné číslo ve formátu RRMMDD/ČČČ(Č)"
+        <div>
+          <mat-button-toggle-group name="isForeigner" [value]="patient.isForeigner">
+            <mat-button-toggle [value]="false" (click)="patient.isForeigner = false;">Jsem občan ČR</mat-button-toggle>
+            <mat-button-toggle [value]="true" (click)="patient.isForeigner = true;">Jsem cizinec</mat-button-toggle>
+          </mat-button-toggle-group>
+        </div>
+      </div>
+
+      <div class="row">
+        <app-form-field label="Rodné číslo *" note="Rodné číslo ve formátu RRMMDD/ČČČ(Č)" *ngIf="!patient.isForeigner"
                         [invalid]="isControlInvalid('personalNumber')" [errors]="getControlErrors('personalNumber')">
           <input formControlName="personalNumber" name="personalNumber" placeholder="Vaše rodné číslo"
-                 mask="000000/0009" [showMaskTyped]="patient.personalNumber.length < 10"
+                 mask="000000/0009" [showMaskTyped]="patient.personalNumber === undefined || patient.personalNumber.length < 10"
                  required appPersonalNumberValidator>
         </app-form-field>
 
+        <app-form-field label="Číslo pojištěnce *" note="Rodné číslo nebo číslo na kartičce zdravotní pojišťovny" *ngIf="patient.isForeigner"
```

Comment:
Dal bych label
`Číslo pojištěnce uvedené na kartičce zdravotní pojišťovny`

Aby bylo jasné, co chceme (RČ je matoucí, u někoho to je stejné, u někoho nikoliv)